src/lib.rs

```diff
@@ -111,7 +104,8 @@ impl Database {
         let rt = runtime(&mut cx)?;
         rt.block_on(async move {
             let db = db.lock().await;
-            db.sync().await
+            let ret = db.sync().await;
+            ret
```

Comment:
What's the purpose of the `ret` variable?

Reply:
none, I just had debug symbols around it earlier, I'm not done cleaning this PR yet :)